sdks/python/apache_beam/utils/counters.py

```diff
@@ -24,39 +24,84 @@
 """
 
 import threading
+
 from apache_beam.transforms import cy_combiners
 
 
+class IOTargetName(object):
+
+  def __init__(self,
```

Comment:
Do you expect users to use this `__init__` method directly?

Perhaps you can use a `namedtuple` and move static methods out of it.